### DISCUSSION TITLE
How to mount folder on another machine

### DISCUSSION THREAD

**Discussion by Ferrangelo** [borgbase/vorta#2058] [FAQ] [answered]
I have just started using vorta, and I have backed up some folders of machine1 in an external hard disk. I can then mount them in machine1. 

I would like to mount, on machine2, the same folder contained in my external hard disk, just in case machine1 fails. Is this possible? If yes, how?

**Comment by m3nu** [comment, marked as answer]
It's possible, just not at the same time.

Simply set up Borg or Vorta on both machines and make sure they can both access the repo. I do this all the time to restore files from NAS servers for example.
